recipes/pcmdi_metrics/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "pcmdi_metrics" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PCMDI/pcmdi_metrics/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a2ed18d379d128cc994d78dabb5bc1da39f3331e26fa8a314705a7cc5fee371f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - compositeDiurnalStatistics.py = pcmdi_metrics.diurnal.scripts.compositeDiurnalStatistics:main
+    - computeStdOfDailyMeans.py = pcmdi_metrics.diurnal.scripts.computeStdOfDailyMeans:main
+    - fourierDiurnalAllGrid.py = pcmdi_metrics.diurnal.scripts.fourierDiurnalAllGrid:main
+    - fourierDiurnalGridpoints.py = pcmdi_metrics.diurnal.scripts.fourierDiurnalGridpoints:main
+    - savg_fourier.py = pcmdi_metrics.diurnal.scripts.savg_fourier:main
+    - std_of_dailymeans.py = pcmdi_metrics.diurnal.scripts.std_of_dailymeans:main
+    - std_of_hourlyvalues.py = pcmdi_metrics.diurnal.scripts.std_of_hourlyvalues:main
+    - std_of_meandiurnalcycle.py = pcmdi_metrics.diurnal.scripts.std_of_meandiurnalcycle:main
+
+requirements:
+  build:
+    - python >=3.6
+    - setuptools
+    - numpy =1.21.3
+  host:
+    - python >=3.6
```

Since this is noarch you only need `host`:
```suggestion
  host:
    - python >=3.6
    - setuptools
```